src/Halogen/HTML/Events.purs

```diff
@@ -260,7 +260,11 @@ touchHandler = unsafeCoerce
 -- | argument of `handler`.
 addForeignPropHandler :: forall r i value. EventType -> String -> (Foreign -> F value) -> (value -> Maybe i) -> IProp r i
 addForeignPropHandler key prop reader f =
```

Comment:
Here is it. Frankly it looks like 4.00 is silently broken due to `unsafeToForeign` :confused: